README.md

```diff
@@ -24,6 +24,52 @@ See also:
 
 Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF). If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Jaeger plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2017/09/13/cncf-hosts-jaeger/).
 
+## Features
+
+### High Scalability
+
+Jaeger backend is designed to have no single points of failure and to scale with the business needs.
+For example, any given Jaeger installation at Uber is typically processing several billions of spans per day.
+
+### Native support for OpenTracing
+
+Jaeger backend, Web UI, and instrumentation libraries have been designed from grounds up to support the OpenTracing standard.
+  * Representing traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
```

Comment:
these three bullet points will read better if they all start in the same way, ie "Represent, support, support"